packages/graphql/src/schema/validation/custom-rules/directives/valid-directive-field-location.ts

```diff
@@ -194,7 +194,7 @@ function validFieldOfInterfaceTypeLocation({
     directiveNode: DirectiveNode;
     parentDef: InterfaceTypeDefinitionNode | InterfaceTypeExtensionNode;
 }) {
-    if (SCHEMA_CONFIGURATION_FIELD_DIRECTIVES.includes(directiveNode.name.value)) {
+    if (SCHEMA_CONFIGURATION_FIELD_DIRECTIVES.includes(directiveNode.name.value as any)) {
```

Comment:
to avoid `as any` here, please use the `isInArray` utility function

```suggestion
    if (isInArray(SCHEMA_CONFIGURATION_FIELD_DIRECTIVES,directiveNode.name.value)) {
```